Documents/panel.md

```diff
@@ -1,17 +1,29 @@
-# Panel CSV Parser
+# Panel CSV Loader
 
 ## Overview
-The panel parser is designed to read a CSV file and load the data onto the HMDA-Platform.  The CSV file should use the `|` (pipe) delimiter, and should include a header row as the first line.
+The panel loader is designed to read a CSV file and load the data onto the HMDA-Platform.  The CSV file should use the `|` (pipe) delimiter, and should include a header row as the first line.
+
+## Environment Variables
+There are two environment variables used by the panel loader.  Both must be set correctly in order for the data to be sent to the admin API.
+
+For testing locally, no changes need to be made.  The defaults for both of these variables will point to the correct local admin API.
+
+For loading panel data into the cluster, navigate to a running `hmda-platform` service in DC/OS.  Under the "Endpoints" section you'll find the IP address and three ports.  As of now, the middle port will always be the admin API.  In your terminal, set both of the environment variables to the DC/OS values.
```

Comment:
I don't think we should be putting DC/OS-specific bits here in the public repo.  If there are tips/tricks to getting it running there, those should probably go in our internal repo's docs.  I think this could just refer to a _remote system_.  Something like:

>When loading panel data into a remote instance of the Admin API, you'll need to set the following environment variables.